File: euler5.rb
i = 2520
while i < 1000000000
	div = 20
	while div > 0
		if i % div != 0
			break
		elsif div == 1
			puts i
			break
		else
			div-=1
		end
	end
	i+=20
	if div == 1
		break
	end
end

File: euler5.c
#include<stdio.h>

int main()
{
  long long number = 2500;
  for (number; number < 240000000; number+= 20)
  { 
    int div = 20;
    while ( div > 0 )
    {
      if (number % div == 0)
      {
        div--;
      }
      else
      {
        break;
      }
     }
  if ( div == 0)
  {
    printf("The answer is %lli\n",number);
    break;
  }
  }
}

